frontend/app/vite.config.ts

```diff
@@ -84,6 +84,28 @@ export default defineConfig({
   server: {
     open: true,
     port: 3000,
+    host: true,
+    proxy: {
+      // You can find all backend endpoints in
+      // https://github.com/streamlit/streamlit/blob/develop/lib/streamlit/web/server/server.py
+      "/_stcore": {
+        target: "http://localhost:8501",
```

Comment:
Do we want to allow 8501 to be overridable by an env variable or something? Or at least a constant in this file in case we want to adjust it?

Reply:
Good idea 👍  I made the full URL configurable via the env variable... which would also allow to connect the dev server to a remote instance.